bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/acqcam/AcqCamGUI.java

```diff
@@ -1,230 +0,0 @@
-/**
- * Title:        JSky<p>
- * Description:  AcqCam Instrument Editor GUI<p>
- * Company:      Gemini<p>
- * @author Allan Brighton
- * @version 1.0
- */
-package jsky.app.ot.gemini.acqcam;
-
-import java.awt.*;
-import javax.swing.*;
-import javax.swing.border.*;
-
-import jsky.util.gui.DropDownListBoxWidget;
-import jsky.util.gui.NumberBoxWidget;
-
-public class AcqCamGUI extends JPanel {
```

Comment:
Good to git rid of stuff we aren't using.  Since this is unrelated to this PR and there are so many deletions, it might be best (in the future) to relegate this type of thing to a separate PR or at least a separate commit.